src/support/action/accountsLogin.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Log in to particular version of nice accounts indepentendly of TopHat
+ * @param  {string}   environment The domain of accounts to use to sign to nice accounts (beta, live or test)
+ * @param  {string}   username The email address used to sign in to Nice Accounts
+ * @param  {string}   password The password used to sign in to Nice Accounts
+ */
+module.exports = (environment, username, password) => {
+	// If you are already logged in
+	if (browser.getCookie("__nrpa_2.2")) {
+		return;
+	}
+
+	const url = ["https://" + environment + "-accounts.nice.org.uk"].join();
```

Comment:
This won't work for live accounts - it's just accounts.nice.org.uk. Also, you don't need join if you're concatenating. It might be better to use ES6 template literaly rather than concatenation - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals